raiden/ui/cli.py

```diff
@@ -306,4 +306,4 @@ def run(ctx, **kwargs):
         gevent.signal(signal.SIGINT, event.set)
         event.wait()
 
-        app_.stop()
+        app_.stop(graceful=True)
```

Comment:
> This may seem to contradict what I wrote above, but I think it doesn't :)

Hmmm no it does indeed contradict what you said :)
We need to discuss what the default user facing behaviour should be and also if we will allow configuration or not.

>as for a fixture: unless testing the graceful shutdown explicitly I don't see why we want tests to wait for cleanup of throwaway blockchain state.

I agree. We should only have it when we want to explicitly test for that scenario. So the default fixture value should be `false. I know it's a bit more work than manually setting it to false in the app constructor in the one test but perhaps in the future there will be more tests or there are gonna be some other fixtures which wrap the app and end up being used in tests in which we would also want to see behaviour during a graceful shutdown.